### ISSUE TITLE
252 Error Code: -1003, JSON formatting error 

### ISSUE THREAD

**Issue by fraxalotl** [apatsufas/homebridge-tapo-p100#6]
**Describe The Bug:**
I installed 1.3.5-beta.0 via npm. I can control the bulb from the home app, (though it shows up twice, One "colorlight", one "light"), and in Homebridge logs I get errors ("`[Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error`")

**To Reproduce:**
`npm install -g homebridge-tapo@1.3.5-beta.0 `

**Expected behavior:**
```
[Tapo Smart Platform] Initializing TapoP100 platform...
[Tapo Smart Platform] Loading accessory from cache: 192.168.0.XX
[Tapo Smart Platform] Adding new accessory: 192.168.0.XX
Homebridge is running on port XXXXX.
```
(No errors)

**Logs:**

```
[12/04/2021, 00:20:22] [HB Supervisor] Starting Homebridge with extra flags: -I
[12/04/2021, 00:20:22] [HB Supervisor] Started Homebridge v1.1.7 with PID: XXXXX
[12/04/2021, 00:20:23] Loaded config.json with 0 accessories and 4 platforms.
[12/04/2021, 00:20:23] ---
...
[12/04/2021, 00:20:24] Loaded plugin: homebridge-tapo@1.3.5-beta.0
[12/04/2021, 00:20:24] Registering platform 'homebridge-tapo.TapoP100'
...
[12/04/2021, 00:20:24] ---
[12/04/2021, 00:20:24] Loading 4 platforms...
[12/04/2021, 00:20:24] [Config] Initializing config platform...
[12/04/2021, 00:20:24] [Config] Running in Service Mode
...
[12/04/2021, 00:20:24] [Tapo Smart Platform] Initializing TapoP100 platform...
[12/04/2021, 00:20:24] [Tapo Smart Platform] Loading accessory from cache: 192.168.0.XX
[12/04/2021, 00:20:24] [Tapo Smart Platform] Adding new accessory: 192.168.0.XX
...
[12/04/2021, 00:20:25] Homebridge is running on port XXXXX.
[12/04/2021, 00:21:35] [Tapo Smart Platform] 371 Error: Request failed with status code 429
[12/04/2021, 00:22:00] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:00] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:02] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:02] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:04] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:04] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:05] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:13] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:13] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:14] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:14] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:17] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:17] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:18] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:18] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:22] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:22] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:22] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:22] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:23] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:23] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:25] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:25] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:25] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:25] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:26] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:27] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:28] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:28] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:28] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:28] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:28] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:28] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:29] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:29] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:30] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:30] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:30] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:30] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:30] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:30] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:31] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:32] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:34] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:34] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:35] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:35] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:40] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:40] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:41] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:41] [Tapo Smart Platform] 252 Error Code: -1003, JSON formatting error 
[12/04/2021, 00:22:44] [Tapo Smart Platform] 68 Error Code: -1008, undefined
```

**Plugin Config:**

```json
{
    "name": "Tapo Smart Platform",
    "username": "XXXXX@XXXXX.com",
    "password": "XXXXXXXXXXXXXXXXXX",
    "devices": [
        {
            "name": "Tapo Bulb L530",
            "host": "192.168.0.XX",
            "type": "Colorlight"
        }
    ],
    "platform": "TapoP100"
}
```
**Environment:**

* **Plugin Version**: Beta: v1.3.5-beta.0
* **Homebridge Version**: v1.1.7
* **Node.js Version**: v15.6.0
* **NPM Version**: v7.4.0
* **Operating System**: Raspbian GNU/Linux Buster (10)

**Comment by apatsufas** [comment]
@fraxalotl Thanks for the bug report. I have published 1.3.6-beta.0 which should fix the error.

**Comment by apatsufas** [comment]
Fixed with 1.4.3
